core/modules/update/update.admin.inc

```diff
@@ -104,8 +104,9 @@ function update_settings_validate($form, &$form_state) {
 /**
  * Form submission handler for update_settings().
  *
- * Also invalidates the cache of available updates if the "Check for updates of
- * disabled modules, themes, and layouts" setting is being changed. The available updates
+ * Also invalidates the cache of available updates if the
+ * "Check for updates of disabled and uninstalled modules, themes, and layouts."
```

Comment:
...funny line wrap there. Unless I'm missing something from the coding standards, then this should be wrapped like so instead:

```
* Also invalidates the cache of available updates if the "Check for updates of
* disabled and uninstalled modules, themes, and layouts." setting is being
* changed. The available updates report needs to refetch available update data
* after this setting changes or it would show misleading things (e.g., listing
* the disabled projects on the site with the "No available releases found"
* warning).
```